packages/babel-types/src/definitions/core.ts

```diff
@@ -37,7 +37,7 @@ defineType("ArrayExpression", {
           assertNodeOrValueType("null", "Expression", "SpreadElement"),
         ),
       ),
-      default: !process.env.BABEL_TYPES_8_BREAKING ? [] : undefined,
+      default: !process.env.BABEL_8_BREAKING ? [] : undefined,
```

Comment:
Can we warn if somebody uses the old one? Like, with this at the beginning of the file:
```js
if (process.env.BABEL_TYPES_8_BREAKING) {
  console.warn("BABEL_TYPES_8_BREAKING is not supported anymore. Use the latest Babel 8.0.0 pre-release instead!");
}